python_modules/dagster/dagster/_core/definitions/assets/graph/remote_asset_graph.py

```diff
@@ -474,20 +474,6 @@ def _asset_check_nodes_by_asset_key(self) -> Mapping[AssetKey, Sequence[RemoteAs
     def get_checks_for_asset(self, asset_key: AssetKey) -> Sequence[RemoteAssetCheckNode]:
         return self._asset_check_nodes_by_asset_key.get(asset_key, [])
 
-    def get_check_keys_for_assets(
```

Comment:
the base class implementation here is better https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/_core/definitions/assets/graph/base_asset_graph.py#L400-L407